ayon_api/entity_hub.py

```diff
@@ -40,7 +40,13 @@ def __init__(
     ):
         if not connection:
             connection = get_server_api_connection()
+        major, minor, _, _, _ = connection.server_version_tuple
+        path_start_with_slash = True
+        if (major, minor) < (0, 6):
```

Comment:
Maybe for another patch, this could be a method of `connection`, something like:
```py
connection.version_smaller_than(0, 6) 
#returns True or False
```

Reply:
Hmm, probably will look into that...